.github/workflows/nightly-test.yaml

```diff
@@ -14,58 +19,86 @@ jobs:
       matrix:
         os: ["ubuntu:20.04", "ubuntu:22.04", "ubuntu:24.04"]
         arch: ["amd64", "arm64"]
-        release: ["latest/edge"]
+        channel: ["latest/edge"]
       fail-fast: false # TODO: remove once arm64 works
```

Comment:
nit: arm64 should work now - we can remove that.

Reply:
The integration tests are still a bit flakey, we could keep ``fail-fast: false`` for now to see if the other jobs pass.